apps/frontend/src/index.css

```diff
@@ -28,18 +28,6 @@
     }
 }
 
-.tooltip__arrow:before {
```

Comment:
@AfeezGL, this pseudo-element is for the tooltip style in the studio. Did you move it elsewhere?

Reply:
@kaf-lamed-beyt No, I removed it, radix-ui already provided a `Tooltip.Arrow` primitive, we don't need to create another tooltip arrow. More on that [here](https://www.radix-ui.com/primitives/docs/components/tooltip#arrow)

Reply:
I see. Thanks for spelling that out!